### PR TITLE
Update README.md - add warning that you need to select all accounts you've used in 2024 due to IBCE to IBIE migration

### PR DESCRIPTION
**Pull request by jamsix**
Add warning that you need to select all accounts you've used in 2024 due to IBCE to IBIE migration.

### CODE REVIEW

**Comment by jamsix** [comment]
@pronebird drop an eye on this please

File: README.md [hunk 1]
```diff
@@ -50,7 +50,10 @@ Odpri datoteko **taxpayer.xml** in vnesi svoje davčne podatke.
...
 
-Ker se je z 2021 zaradi Brexita večino evropskih IBKR računov premaknilo (oziroma realno so se računi ukinili in novi kreirali) iz Londona v eno izmed evropskih podružnic lahko naletiš na težave pri izvozu podatkov. Na strani **Reports** je desno od tega v modrem še številka tvojega računa. Če jo klikneš se odpre meni kjer lahko izbiraš za kateri račun želiš poročila, po defaultu sicer ne kaže ukinjenih računov, vendar če klikneš na filter lahko to spremeniš, da se pokaže tudi ukinjen račun. Potem pa obkljukaš še tistega in klikneš **Continue**. Sedaj boš poleg naslova **Reports** videl dve številki računov, in tudi podatki bodo iz obeh.
+**Pozor**:
+Pred 2021 smo slovenske stranke IBKR imele račun na britanski podružnici IBUK. 2021 so bili naši računi večinoma prenešeni na madžarsko podružnico IBCE, v sredini leta 2024 pa iz madžarske na irsko podružnico IBIE. Pri migraciji smo uporabniki obdržali uporabniško ime, toda dobili smo nov IBKR račun (Uxxxxxxx). Zato je pomembno, da pri izdelavi poročila izberemo vse račune, ki so bili aktivni v izbranem letu.
```
Comment (pronebird) [inline README.md:54]: Nit: Se mi zdi preneseni (s namesto š) je bolj ustrežna oblika ampak slovenščina je moj tretji ali četrti jezik 🤣

**Review by pronebird — APPROVED** [review]
Hvala!

File: README.md (hunk 1 above)
Comment (jamsix) [reply README.md:54]: Oboje gre, ampak če prav razumem (https://svetovalnica.zrc-sazu.si/topic/4168/dele%C5%BEni%C5%A1ke-oblike-prenesen-prene%C5%A1en-in-sorodne-zagate) imaš prav :)

**Pull request merged**